app/graphql/enums/digital_object/order_fields_enum.rb

```diff
@@ -0,0 +1,7 @@
+# frozen_string_literal: true
+
+class Enums::DigitalObject::OrderFieldsEnum < Types::BaseEnum
+  value 'RELEVANCE', 'sorting by relevance score', value: 'score'
+  value 'LAST_MODIFIED', 'sorting by last modified at', value: 'updated_at_dtsi'
+  value 'TITLE', 'sorting by non-sort title', value: 'sort_title_ssi'
```

Comment:
Updated comment:

Seeing this capital enum made me look up whether or not enums should be capitalized with GraphQL. And yes, they should! http://spec.graphql.org/June2018/#sec-Enum-Value

So you have that right, but we may want to update some of our other enums that aren't all-caps values.

Reply:
I'll create a ticket about updating the Enums to caps and I can do that work in a subsequent PR.

Yea, I don't love putting the solr keys there but I don't have a great suggestion for how to decouple it. This problem will come up in other places too (filters, etc). I would rather leave it as is for now and iterate on it later.

Reply:
RE: Solr keys problem, I got a few ideas last night but I would like to see where all the seams are before deciding on an approach.

Reply:
👍